.gitmodules

```
@@ -0,0 +1,3 @@
+[submodule "lib/pico-sdk"]
+	path = lib/pico-sdk
+	url = https://github.com/raspberrypi/pico-sdk.git
```

```suggestion
	path = lib/pico-sdk
	url  = ../../raspberrypi/pico-sdk.git
```
You can do relative URLs so that this works with either HTTPS or SSH.